scripts/us_census/pep/us_pep_sexrace/postprocess.py

```diff
@@ -0,0 +1,370 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Script to read all
+the generated csv for
+National, State and County
+and generate final csv, MCF, TMCF file
+"""
+
+import pandas as pd
+import os
+
+_CODEDIR = os.path.dirname(os.path.realpath(__file__))
+
+
+def create_single_csv(output_files_names):
```

Comment:
The following 2 functions (generate_mcf() and generate_tmcf()) uses pytype, can we be consistent and use pytype for all functions?

Reply:
Done the changes.